newLint.sh

```diff
@@ -17,18 +18,32 @@ if [ $# -eq 1 ]; then
     exit 1
 fi
 
-if [ -e lint_$1.go ]
+if [ $# -eq 2 ]; then
+    echo "Not enough arguments provided..."
+    echo "$USAGE"
+    exit 1
+fi
+
+if [ ! -d lints/$1 ]
 then
-   echo "File already exists. Can't make new file."
+   echo "Directory $1 does not exist. Can't make new file."
    exit 1
 fi
 
-FILENAME=$1
-TESTNAME=$2
 
-cp template lints/lint_$FILENAME.go
+if [ -e lints/$1/lint_$2.go ]
+then
+   echo "File already exists. Can't make new file."
+   exit 1
+fi
+
+PATHNAME=$1
+LINTNAME=$2
+FILENAME=${LINTNAME:2}
```

Comment:
Am I correct in understanding the `LINTNAME`/`FILENAME` split here is being used to remove the `e_`, `w_` etc prefix from lint filenames? If so can you add a comment to that effect? I think it's a little bit subtle as is.

Reply:
Yes you are right. I added one comment regarding this.